syncd/Syncd.cpp

```diff
@@ -563,6 +566,30 @@ sai_status_t Syncd::processObjectTypeGetAvailabilityQuery(
     return status;
 }
 
+sai_status_t Syncd::processLinkEventDampingConfigSet(
+        _In_ const swss::KeyOpFieldsValuesTuple &kco)
+{
+    SWSS_LOG_ENTER();
+
+    sendLinkEventDampingConfigResponse(SAI_STATUS_NOT_IMPLEMENTED);
```

Comment:
you are sending response every time, but SET you implemented is not always synchronous, you added (m_syncMode), so actually sairedis will receive response even in async mode which will break

Reply:
Thank you for pointing that out. Done.